metadata/llm_metadata.yml

```
@@ -129,4 +129,76 @@ gemini:
+    presence_penalty:
+      type: number
+      default: 0.0
+      minimum: -2.0
+      maximum: 2.0
+      description: "The presence_penalty hyperparameter penalizes the model for generating words that are not present in the context or input prompt."
+    frequency_penalty:
+      type: number
+      default: 0.0
+      minimum: -2.0
+      maximum: 2.0
+      description: "The frequency_penalty hyperparameter penalizes the model for generating words that have already been generated in the current response."
```

yamllint

[error] no new line character at the end of file

(new-line-at-end-of-file)